public/src/js/widgets/columns/packages.js

```diff
@@ -121,16 +122,56 @@ export default class Package extends ColumnWidget {
         .catch(() => {});
     }
 
+    savePackageEdits(parent, index, storyPackage) {
+        var name = storyPackage.meta.name().trim();
+        if (name.length < 3) {
+            alert('Package name needs to include at least three characters');
+            return;
+        }
+
+        return authedAjax.request({
+            url: '/story-packages/edit/' + storyPackage.id,
+            type: 'post',
+            data: JSON.stringify({
+                name: name,
+                isHidden: parent.baseModel.priority === 'training'
+            })
+        })
+        .then((response) => {
+
+            var storyPackage = new StoryPackage(response);
+            var results = parent.searchResults();
```

Comment:
I don't like the logic here, children shouldn't change things on `parent.`. It's better if the parent listens to `update:package` and updates the search results in the callback